flow/designs/asap7/riscv32i/config.mk

```diff
@@ -24,7 +24,7 @@ export CORE_AREA = 5 5 75 85
 
 export PLACE_DENSITY_LB_ADDON = 0.10
 
-export PLACE_PINS_ARGS    = -exclude left:* -exclude right:* -exclude top:*
+export IO_CONSTRAINTS     = $(DESIGN_HOME)/$(PLATFORM)/$(DESIGN_NICKNAME)/io.tcl
```

Comment:
Are we dropping PLACE_PINS_ARGS? Why the preference on IO_CONSTRAINTS=io.tcl?

Non-sequitor: Renaming IO_CONSTRAINTS to IO_TCL would be more consistent with the other variable names CONSTRAINTS_SDC, MACRO_PLACEMENT_SDC.